content/articles/testing-for-broken-authentication-in-web-apps/index.md

```diff
@@ -0,0 +1,117 @@
+Authentication is a mechanism put in place to determine if a user is who they say they are, through either a username and password-based system or any other form of authentication system.
+
+### What is Broken Authentication?
+
+Broken Authentication is a type of vulnerability that allows attackers to get into a web application without proper credentials. This could be carried out either by bypassing the authentication mechanism put in place or brute=forcing another user's account. If the attacker successfully bypasses or brute-forces his way into another user's account, they gain access to all the data and privileges of that user account.
+
+According to the [OWASP Top 10 2021 report](https://owasp.org/Top10/A07_2021-Identification_and_Authentication_Failures/), Broken Authentication is ranked number 7 and grouped under `Identification and Authentication Failures`. 
```

Comment:
Add a little more about the report would definitely help - any insights from data or any new information that is relevant to this topic